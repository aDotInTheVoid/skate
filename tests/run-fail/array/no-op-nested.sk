// TODO: Fix how we display these

fn main() {
    [[[1]]]  + [[[2]]]
}