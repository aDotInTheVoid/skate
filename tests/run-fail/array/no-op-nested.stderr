error: Unknown binop `+`, for `array` and `array`
  ┌─ $DD/tests/run-fail/array/no-op-nested.sk:4:14
  │
4 │     [[[1]]]  + [[[2]]]
  │     -------  ^ ------- LHS evaluated to `[[[2]]]`
  │     │        │  
  │     │        In this operator
  │     LHS evaluated to `[[[1]]]`

